src/MudBlazor.Markdown/MudMarkdown.razor.cs

```diff
@@ -432,6 +432,19 @@ protected virtual void RenderList(ListBlock list, RenderTreeBuilder builder)
 						builder.CloseElement();
 						break;
 					}
+					case FencedCodeBlock code:
+					{
+						builder.OpenElement(ElementIndex++, "li");
+						var text = code.CreateCodeBlockText();
+
+						builder.OpenComponent<MudCodeHighlight>(ElementIndex++);
+						builder.AddAttribute(ElementIndex++, nameof(MudCodeHighlight.Text), text);
+						builder.AddAttribute(ElementIndex++, nameof(MudCodeHighlight.Language), code.Info ?? string.Empty);
+						builder.AddAttribute(ElementIndex++, nameof(MudCodeHighlight.Theme), CodeBlockTheme);
+						builder.CloseComponent();
+						builder.CloseElement();
+						break;
+					}
```

Comment:
First of all thank you for making the PR.

But sorry. this implementation is just copy paste from lines 178-182 and it cannot be accepted. Please do the following:
1) create a function `RenderFencedCodeBlock(RenderTreeBuilder, FencedCodeBlock)` and use it on the line 176
```cs
// 176
case FencedCodeBlock code:
{
	RenderFencedCodeBlock(builder, code);
	break;
}

// 464 (or somewhere else at the end)
protected virtual void RenderFencedCodeBlock(in RenderTreeBuilder builder, in FencedCodeBlock code)
{
	var text = code.CreateCodeBlockText();

	builder.OpenComponent<MudCodeHighlight>(ElementIndex++);
	builder.AddAttribute(ElementIndex++, nameof(MudCodeHighlight.Text), text);
	builder.AddAttribute(ElementIndex++, nameof(MudCodeHighlight.Language), code.Info ?? string.Empty);
	builder.AddAttribute(ElementIndex++, nameof(MudCodeHighlight.Theme), CodeBlockTheme);
	builder.CloseComponent();
}
```

2) Instead of your code please use this (there is no need to create a new `li` element for the code block, at least this is how it should work in your example):
```cs
case FencedCodeBlock x:
{
	RenderFencedCodeBlock(builder, x);
	break;
}
```

Then it will look a bit similar to what is rendered on GitHub. However, the line "Replace `username` .." should not have the "li" tag, so if you can also fix this problem it would be nice. If you do not want to do it, then I can do it in a separate PR. To give you a hint - it can be determined by "Parent.Order" properties of ParagraphBlock or FencedCodeBlock is the parent is `ListItemBlock`

![image](https://github.com/MyNihongo/MudBlazor.Markdown/assets/47413092/a2383dd8-20db-4e68-801e-d45bc12247e5)